.github/actions/setup-import-system-symbols/action.yml

```diff
@@ -39,9 +39,6 @@ runs:
     - name: "Install Python dependencies"
       shell: bash
       run: python3 -m pip install --user -r requirements.txt
-    - name: Select Xcode version
```

Comment:
macOS 11 and macOS 13 don't have Xcode 13. I think we don't need to specify the Xcode version. We can just use the default of the runner.